src/eduid_common/api/utils.py

```diff
@@ -154,21 +154,32 @@ def static_url_for(f):
     return app
 
 
-def verify_relay_state(relay_state, safe_default='/'):
+def verify_relay_state(relay_state, safe_default='/', **kwargs):
```

Comment:
Can we have explicit args with None as default instead of the kwargs? Don't want to end up where pysaml2 is...

Reply:
I had missed this, fixed